backend/code_coverage_backend/gcp.py

```diff
@@ -368,7 +370,7 @@ def get_suites(self, repository):
         suites = self.redis.smembers(KEY_SUITES.format(repository=repository))
         return sorted(map(lambda x: x.decode("utf-8"), suites))
 
-    def ingest_available_reports(self, repository):
+    def ingest_available_reports(self, repository, until=None):
```

Comment:
How are you planning to use this?

Reply:
Just once either from my computer or from the heroku CLI run command:

```python
from code_review_backend import taskcluster
from datetime import timedelta
taskcluster.auth()
taskcluster.load_secrets("project/relman/code-coverage/runtime-testing", prefixes=["common"])
c = GCPCache()
c.ingest_available_reports("mozilla-central", until=timedelta(days=30))
```

This is just a shortcut to load only reports from last month without loading all the others

Reply:
Maybe we should add this as a script too